hazelcast/src/main/java/com/hazelcast/internal/management/events/EventMetadata.java

```diff
@@ -45,7 +45,10 @@ public EventMetadata(EventType type, long timestamp) {
         WAN_CONFIGURATION_ADDED(8),
         ADD_WAN_CONFIGURATION_IGNORED(9),
         WAN_SYNC_IGNORED(10),
-        WAN_CONFIGURATION_EXTENDED(11);
+        WAN_CONFIGURATION_EXTENDED(11),
+        RELOAD_STARTED(12),
+        RELOAD_PROGRESS(13),
+        RELOAD_FINISHED(14);
```

Comment:
is there any chance that config reload fails?
Can it be started in parallel by few members?

Reply:
https://github.com/hazelcast/hazelcast/pull/20260/commits/6f5cd7314b6798cd8104c74dcf023a432b40b8e5

Note that with this commit, progress events also include their object namespace.